src/app/test-record/visit/visit-edit/visit-edit.component.scss

```diff
@@ -0,0 +1,10 @@
+#test-testStationType {
+  border: none;
+}
+#test-testStationType:focus {
+  outline: none;
+  box-shadow: none;
+}
+#test-testStationType:disabled {
+  color: #0b0c0c;
```

Comment:
same for scss
Also i would be very careful to use and id for it, a class should be enough, we don't need to increase specificity for nothing,
We could probably port those variables over to a separate scss file if we can't use govuk but i suspect we should be able to with the appropriate markup in place